tests/testthat/helper-indexing.R

```diff
@@ -0,0 +1,24 @@
+mm <- function(...) {
+  v <- as.numeric(as.vector(list(...)))
+  matrix(v, nrow = sqrt(length(v)))
+}
+
+am <- function(x) {
```

Comment:
am() is a helper function in tests. maybe it could be called as_matrix? @szhorvat

Reply:
How about `canonicalize_matrix`? (Since you asked. But use anything you like.)

Reply:
Thank you!